### PR TITLE
Fix build errors

### PR DESCRIPTION
**Pull request by afd**

### CODE REVIEW

**Review by paulthomson — APPROVED** [review]

**Comment by Vasniktel** [comment]
There is also a bug in https://github.com/KhronosGroup/SPIRV-Tools/blob/4c239bd81bf79337e4a0cfe1c9da9caf1b25108a/test/fuzz/transformation_move_instruction_down_test.cpp#L610 if I recall correctly.

File: source/fuzz/transformation_move_instruction_down.cpp [hunk 1]
```diff
@@ -14,9 +14,9 @@
 
 #include "source/fuzz/transformation_move_instruction_down.h"
 
-#include "external/spirv-headers/include/spirv/unified1/GLSL.std.450.h"
 #include "source/fuzz/fuzzer_util.h"
 #include "source/fuzz/instruction_descriptor.h"
+#include "spirv/unified1/GLSL.std.450.h"
```
Comment (Vasniktel) [inline source/fuzz/transformation_move_instruction_down.cpp:19]: Thanks for fixing!

Comment (afd) [reply source/fuzz/transformation_move_instruction_down.cpp:19]: No problem!

**Comment by Vasniktel** [comment]
It seems tests are failing because of the assertion in the fact manager. https://github.com/KhronosGroup/SPIRV-Tools/blob/51b826140a63b74c93cfe8074ef643dec026adb4/source/fuzz/transformation_flatten_conditional_branch.cpp#L581-L582 should be moved after the call to `InvalidateAnalyses`.

Alternatively, you could comment this out as well as disable the corresponding tests and I will resolve this issue when I refactor the fact manager as in #3698.

**Comment by afd** [comment]
Thanks - I am working on a fix.  Seems like these PRs landed in an unfortunate order :)

**Pull request merged**